apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html

```diff
@@ -277,6 +277,12 @@ <h6>
           <div id="help-text-total_claims_value_${entities}" class="help-text">
           </div>
         </div>
+        ${ entities === 5 ?
+            {% if not r %}
+             '<h6>If you need to associate more than 5 entities with your registration, please <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</h6>'
```

Comment:
This message is not a [heading (`<h1>–<h6>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)). Please use a more appropriate tag.

<details><summary>Styling vs. Semantics</summary>

Choose tag based on semantics, not how it looks when it renders; looks are what CSS is for.

If the most appropriate tag does not have styling suitable to the context, then consider:
- the [message UI component (`c-message`)](https://tacc.utexas.edu/static/ui/components/detail/c-message--type-cms.html)
	<sup>Requires `{% include 'snippets/tup-175-css-alerts-messages-ui-pattern.html' %}` to get `c-message` CSS.</sup>
- a new global class and style for APCD
    <sup>(added to `apcd-cms/src/taccsite_custom/apcd-cms/static/apcd-cms/css/site.cms.css`, perhaps `instructions`)</sup>

</details>

<details><summary>Suggestions</summary>

[**`<p>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)
```suggestion
             '<p class="c-message c-message--type-info c-message--scope-inline">If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</p>'
```

[**`<aside>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)
```suggestion
             '<aside class="instructions">If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</aside>'
```

[**`<small>`**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small)
```suggestion
             '<small>If you need to associate more than 5 entities with your registration, <a href="/workbench/dashboard" target="_blank">submit a ticket</a> with your additional entries and your registration ID (displayed after submitting this form).</small>'
```

</details>